pommerman/envs/graphics.py

```diff
@@ -0,0 +1,175 @@
+
+        glEnable(GL_BLEND)
+        glBlendFunc(GL_SRC_ALPHA, GL_ONE_MINUS_SRC_ALPHA)
+
+        @self.window.event
+        def close(self):
+            self.window.close()
+            self.isopen = False
+
+    def set_agent_view_size(self, view_size):
+        self._agent_view_size =view_size
+
+    def set_frame_section(self, frames):
+        self._frame_section = frames
+    
+    def clear_frame_section(self):
+        self._frame_section = []
+
+    def set_agents_state(self, agents):
+        self._agents = agents
+
+    def set_frames_from_game_state(self, state):
+        images = empty_frame(self._board_size)
+
+        for r in range(self._board_size):
+            for c in range(self._board_size):
+                item_value = state[r][c]
+                image = self._sprite_manager.get_image_from_state_value(item_value)
+                images[r][c] = image
+        
+        self.set_frame_section(images)
+
+    def render(self):
+        self.window.clear()
+        self.window.switch_to()
+        self.window.dispatch_events()
+        
+        self.render_main_board()
+        self.render_agents_board()
+        
+        self.window.flip()
+
+    def render_main_board(self):
+        sections = self._frame_section
+
+        for r in range(self._board_size):
+            for c in range(self._board_size):
+                x = c * self._tile_size
+                y = r * self._tile_size
+                sections[r][c].blit(x, y, width=self._tile_size, height=self._tile_size)
+
+    def render_agents_board(self):
+        x_offset = self._board_size * self._tile_size
+        for agent in self._agents:
+            y_offset = agent.agent_id * (self._agent_tile_size * self._board_size)
+            self.render_agent_board(agent, x_offset, y_offset)
+
+        
+
+    def render_agent_board(self, agent, x_offset, y_offset):
+        row, col = agent.position
+        agent_view_size = self._agent_view_size
+        sections = self._frame_section
+        fog = self._sprite_manager.get_fog()
+
+        for r in range(self._board_size):
+            for c in range(self._board_size):
+                x = c * self._agent_tile_size + x_offset
+                y = r * self._agent_tile_size + y_offset
+                if self._is_partially_observable and not all([
+                            row >= r - agent_view_size, row < r + agent_view_size,
+                            col >= c - agent_view_size, col < c + agent_view_size
+                    ]):
+                    fog.blit(x, y, width=self._agent_tile_size, height=self._agent_tile_size)
+                else:
+                    sections[r][c].blit(x, y, width=self._agent_tile_size, height=self._agent_tile_size)
+
+    def close(self):
+        self.window.close()
+        self.isopen = False
+
+    def window_closed_by_user(self):
+        self.isopen = False
+
+    
+class SpriteManager(object):
+
+    def __init__(self):
+        index_images()
```

Comment:
I don't see a reason for `index_images` to be its own function outside, is there any?